PersonApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -178,16 +182,34 @@ public void GetPersonByIdExceptionThrow()
         public async Task UpdatePersonSuccessfulUpdates()
         {
             // Arrange
-            var constructRequest = ConstructUpdatePerson();
-            await _dynamoDb.SaveAsync(constructRequest.ToDatabase()).ConfigureAwait(false);
-            constructRequest.Surname = "Update";
-
+            var constructPerson = ConstructUpdatePerson();
```

Comment:
There is no need to create a 'ConstructUpdatePerson' method, you could use the one for creating a person